core/src/main/java/com/graphhopper/routing/ev/IntEncodedValueImpl.java

```diff
@@ -212,100 +211,7 @@ public final String getName() {
 
     @Override
     public final String toString() {
-        return getName() + "|version=" + getVersion() + "|bits=" + bits + "|min_value=" + minValue
-                + "|negate_reverse_direction" + negateReverseDirection + "|index=" + fwdDataIndex
-                + "|shift=" + fwdShift + "|store_both_directions=" + storeTwoDirections;
-    }
-
-    @Override
-    public boolean equals(Object o) {
-        if (this == o) return true;
-        if (o == null || getClass() != o.getClass()) return false;
-        IntEncodedValueImpl that = (IntEncodedValueImpl) o;
-        return fwdDataIndex == that.fwdDataIndex &&
-                bwdDataIndex == that.bwdDataIndex &&
-                bits == that.bits &&
-                maxValue == that.maxValue &&
-                minValue == that.minValue &&
-                fwdShift == that.fwdShift &&
-                bwdShift == that.bwdShift &&
-                fwdMask == that.fwdMask &&
-                bwdMask == that.bwdMask &&
-                negateReverseDirection == that.negateReverseDirection &&
-                storeTwoDirections == that.storeTwoDirections &&
-                Objects.equals(name, that.name);
-    }
-
-    @Override
-    public final int hashCode() {
-        return getVersion();
-    }
-
-    @Override
-    public int getVersion() {
-        int val = Helper.staticHashCode(name);
-        val = 31 * val + (storeTwoDirections ? 1231 : 1237);
-        val = 31 * val + (negateReverseDirection ? 13 : 17);
-        return staticHashCode(val, fwdDataIndex, bwdDataIndex, bits, minValue, maxValue, fwdShift, bwdShift, fwdMask, bwdMask);
-    }
-
-    /**
-     * Produces a static hashcode for an integer arrays that is platform independent and still compatible to the default
-     * of openjdk.
-     *
-     * @see Arrays#hashCode(int[])
-     */
-    static int staticHashCode(int... vals) {
-        if (vals == null)
-            return 0;
```

Comment:
no need for these methods here anymore, but if you think they are generally useful we could move them into Helper or something?

Reply:
They are gone.